ev3dev2/led.py

```diff
@@ -346,3 +385,231 @@ def all_off(self):
 
         for led in self.leds.values():
             led.brightness = 0
+
+    def reset(self):
+        """
+        Put all LEDs back to their default color
```

Comment:
Does this need to also reset the animation thread?

Reply:
Good catch will add that.